lang/python/python-zmq/patches/fix-detect.patch

```diff
@@ -0,0 +1,31 @@
+Index: pyzmq-22.2.1/buildutils/detect.py
```

Comment:
Since the libzmq version check is disabled (with `skip_check_zmq = True` in setup.cfg), is this patch still necessary?